mkdocs.yml

```
@@ -121,7 +144,7 @@ markdown_extensions:
   - pymdownx.smartsymbols:
       fractions: false
   - pymdownx.snippets:
-      base_path: 'ansible_collections/arista/avd/'
+      base_path: './'
```

https://facelessuser.github.io/pymdown-extensions/extensions/snippets/#specifying-snippet-locations

it is a list now, but string still accepted as documented
```suggestion
      base_path:
        - './'
```